static/src/stylesheets/module/identity/upsell/_layout.scss

```diff
@@ -105,6 +102,11 @@
             }
         }
     }
+    .identity-upsell-block--half-width {
+        @include mq(tablet) {
+            width: 50%;
```

Comment:
I think this could be `max-width: gs-span(6);` instead.  This will make it easier to maintain (less magic numbers) and more consistent with other widths in this file.

Disclaimer: 6 might not the right number of spans 😄